### ISSUE TITLE
alert module shows notification twice

### ISSUE THREAD

**Issue by Quarx64** [MichMich/MagicMirror#1615]
my code is:
self.sendNotification("SHOW_ALERT", {type: "notification", title: "New card detected"});
this occures only after starting the mirror.
after the 2nd or 3rd alert it's OK and i get only one bubble alert on top of the screen.

all modules are updated to version 2.6.0

**Comment by MichMich** [comment]
Please refer to http://forum.magicmirror.builders for support and troubleshooting.

I'm closing this issue for now. If you feel this issue should remain open, please let me know.
